src/system/PageLockModule/Entity/Repository/PageLockRepository.php

```diff
@@ -76,7 +76,7 @@ public function getActiveLocks($lockName, $sessionId)
 
         $query = $qb->getQuery();
 
-        $locks = $query->getResult(\Doctrine\ORM\AbstractQuery::HYDRATE_ARRAY);
+        $locks = $query->getArrayResult();
```

Comment:
is there a difference between these? I've never understood this *hydrating* business.